src/views/index.njk

```diff
@@ -1,21 +1,153 @@
 {% extends "base.njk" %}
 
 {% block content %}
-    <h1>
-        <a href="https://the-federation.info" style="color:inherit">
-            <img src="/images/the-federation.png">
-            <span style="padding-left: 30px;">
-                the federation - a statistics hub
-            </span>
-        </a>
-    </h1>
+<main>
+    <header class="main-header">
+        <div class="main-title">
+            <h1>The Federation</h1>
+            <h2>Welcome to the new social web</h2>
+        </div>
+        <div class="flex">
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                4 <strong>Projects</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                {{ data.total_nodes }} <strong>Nodes</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                {{ data.total_users }} <strong>Users</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                ONE <strong>Network</strong>
+                </div>
+            </div>
+        </div>
+    </header>
 
-    {% include "_network_stats.njk" %}
-    {% include "_node_list.njk" %}
-    {% include "_info_the_federation.njk" %}
-    {% include "_info_site.njk" %}
-
-    <p class="text-center">
-        built with &lt;3 by <a href="https://jasonrobinson.me">jason robinson</a> and <a href="https://github.com/jaywink/the-federation.info/graphs/contributors">contributors</a>
-    </p>
+    <section class="tile">
+        <header>
+            <h2>What is The Federation?</h2>
+        </header>
+        <div>
+            <div class="flex">
+                <div class="col2">
+                    <p>
+                        The Federation refers to a global social network composed by nodes talking together.
+                        Each of them is an installation of a software which supports the <a href="https://github.com/diaspora/diaspora_federation">diaspora* protocol</a>.
```

Comment:
Maybe link to [this](https://diaspora.github.io/diaspora_federation/) instead? It's a bit more descriptive than the code repo.